lib/phoenix_starter/accounts/user.ex

```diff
@@ -0,0 +1,34 @@
+defmodule PhoenixStarter.Accounts.User do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @fields [:email, :name, :password]
+  @required_fields [:email, :name, :password]
+
+  schema "users" do
+    field :email, :string
+    field :password, :string, virtual: true
+    field :encrypted_password, :binary
+    field :name, :string
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(user, attrs) do
+    user
+    |> cast(attrs, @fields)
+    |> validate_required(@required_fields)
+    |> unique_constraint(:email, downcase: true)
+    |> encrypt_password()
+  end
```

Comment:
Could we add a email regex validation and a minimal password size validation ? Even if they're not perfect its still a little more secure

Reply:
Did that. Min 6 chars